api/build.gradle

```
@@ -0,0 +1,37 @@
+plugins {
+    id 'java'
+    id 'org.springframework.boot' version '3.3.1'
```

```suggestion
    id 'java'
    id 'org.springframework.boot' version '3.3.1'
```
왜 자바 플러그인은 버전이 없고, 스프링 플러그인은 버전이 있을까요?

---

Core 플러그인과 External 플러그인의 차이입니다.
그래들은 코어플러그인을 자체 번들로 제공합니다.
하지만 스프링이라던가, 스팟레스와 같은 플로그인의 경우 플러그인 포털과 같은 외부 저장소에서 제공합니다.
외부 저장소에서 제공하기에, 버전을 두어 따로 관리합니다.